content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -45,8 +45,7 @@ value for this field is `600`, i.e. ten minutes.
 Once created, a CertificateSigningRequest must be approved before it can be signed.
 Depending on the signer selected, a CertificateSigningRequest may be automatically approved
 by a {{< glossary_tooltip text="controller" term_id="controller" >}}.
-Otherwise, a CertificateSigningRequest must be manually approved either via the REST API (or client-go)
-or by running `kubectl certificate approve`. Likewise, a CertificateSigningRequest may also be denied,
+Otherwise, a CertificateSigningRequest must be manually approved either via the REST API - the most common methods for making these API requests include kubectl (by running `kubectl certificate approve`) and the Kubernetes go client library [client-go](https://github.com/kubernetes/client-go). Likewise, a CertificateSigningRequest may also be denied,
```

Comment:
```suggestion
Otherwise, a CertificateSigningRequest must be manually approved either via the REST API - the most common
methods for making these API requests include kubectl (by running `kubectl certificate approve`)
and the Kubernetes go client library [client-go](https://github.com/kubernetes/client-go). Likewise, a CertificateSigningRequest may also be denied,
```

Reply:
Please wrap the lines and follow contributor [styleguide](https://kubernetes.io/docs/contribute/style/style-guide/) for contribution rules.

Reply:
This change doesnt add any improvement in the content. However it looks like the sentence is modified to say the exactly same thing.